### PR TITLE
Cohort list

### PR DESCRIPTION
**Pull request by korgan00**
## Changes

Fix https://github.com/Qiskit/saiba/issues/332

## Implementation details

Depends on https://github.com/Qiskit/saiba/pull/348

## How to read this PR

go to `/account/cohort/057c74fa-2efd-4a49-b245-c81bb44142a4/edit`

### CODE REVIEW

File: components/cohort/Edit.vue [hunk 1]
```diff
@@ -75,6 +76,15 @@
         Anyone with url can access
       </bx-checkbox>
     </div>
+    <div class="cohort-edit__action-bar">
+      <bx-btn size="md" kind="ghost" @click="handleCancelRequest">
+        Cancel
+      </bx-btn>
+      <bx-btn size="md" icon-layout @click="handleSendRequest">
```
Comment (techtolentino) [inline components/cohort/Edit.vue:83]: small comment:

the design seems to show that there should be no spacing between the buttons. Can we add a little styling to keep the button set together?


https://user-images.githubusercontent.com/6276074/205134291-e5ae55cd-3173-40a2-bdf6-02c85bfe9fe2.mov

**Review by techtolentino — CHANGES_REQUESTED** [review]
This looks great overall, @korgan00 

I left a couple of minor comments.

Thank you 👏🏼

File: components/cohort/Edit.vue [hunk 2]
```diff
@@ -86,36 +96,52 @@ import 'carbon-web-components/es/components/button/index';
...
-    e: 'changed',
-    data: {
-      cohort: Cohort;
-    }
+    e: 'sendRequest',
+    data: { cohort: Cohort; deletedMembersList: Member[] }
   ): void;
 }>();
 
+const informationMessage = reactive({
```
Comment (yoshiemuranaka) [inline components/cohort/Edit.vue:123]: I think this is used for inline validation messages? could we rename to

```suggestion
const inlineValidationMessage = reactive({
```

File: components/cohort/Edit.vue [hunk 3]
```diff
@@ -177,12 +203,14 @@ async function handleFile(payload: Event) {
 function handleAddMemberButton() {
   if (isValidEmail(addMemberInputText.value)) {
     const isMemberAdded = addMember(addMemberInputText.value);
+
```
Comment (yoshiemuranaka) [inline components/cohort/Edit.vue:206]: Should we consider using a validation library? might clean up a lot of these manual checks

**Review by yoshiemuranaka — COMMENTED** [review]
looks good! left a few comments

File: components/cohort/MembersList.vue [hunk 4]
```diff
@@ -1,15 +1,31 @@
 <template>
   <div class="cohort-member-list">
     <div
-      v-for="(member, index) in members"
+      v-for="email in invitations"
+      :key="email"
+      class="cohort-member-list__item"
+    >
+      i -
```
[the remark this answers is not in the document]
Comment (korgan00) [reply components/cohort/MembersList.vue:8]: Yes, in the next PR i made a minor change in the next PR [changing `i - ` to (invitation)](https://github.com/Qiskit/saiba/pull/383/files#diff-889e6228b5198c39789a87ffa89d7aaa13a81d0d203e3000af700dc43c2f0629) and waiting for a UX approval or change.

[the remark this answers is not in the document]
Comment (korgan00) [reply components/cohort/MembersList.vue:8]: There is new UI update that will be done after closing this PR and the one connecting with services. I think we can resolve this in favor of the redesign.

File: components/cohort/Edit.vue (hunk 3 above)
Comment (korgan00) [reply components/cohort/Edit.vue:212]: I was looking for form validation libraries. Vue has few recommendations and the one that I liked more is this one: [vuelidate](https://vuelidate-next.netlify.app/)
We can move this to a discussion so every dev can give their opinion.

Right now there is few checks, so I think we can delay changing this.

Comment (yoshiemuranaka) [reply components/cohort/Edit.vue:212]: Created #401 to track

**Review by yoshiemuranaka — APPROVED** [review]
ty!

**Pull request merged**